Document modernized usage charts for usage and spend alerts M2 (#2812)
wandb/docs: platform/app/settings-page/billing-settings.mdx

#### Current cycle compared to historical views

The **Current cycle** view is *cumulative*: each bar adds to the previous one to show your running total usage since the start of the current billing period. Use this view to see how your usage is pacing against its allowance. Granularity depends on the length of your billing cycle. For example, a monthly cycle shows daily bars and an annual cycle shows monthly bars.

The **Last 30 days**, **Last 90 days**, **Last 12 months**, and **Last 24 months** views are *historical*: each bar shows the usage for a single day or month rather than a running total. Use these views to compare usage across past periods. Because the top five teams are recalculated for the selected range, the colored teams can differ from those in the current cycle.

Storage is always cumulative, because it measures the total volume currently stored (a 30-day weighted average, in GB) rather than usage consumed during a period.

{/* Source: https://wandb.ai/account-settings/wandb/alerts/storage (multi-tenant, Last 12 months, All teams) */}
<Frame>

>     <img src="/images/app_ui/billing_usage_chart_12_months.png" alt="Storage usage over time chart for the last 12 months, showing one bar per month segmented by team" />

Author: ```suggestion
    <img src="/images/app_ui/billing_usage_chart_12_months.png" alt="Storage usage over time for the last 12 months, measured as a 30-day weighted average, expressed in GB with one bar per month segmented by team" />
```

For questions about usage or billing, contact your AISE or [Support](mailto:support@wandb.com).

### View usage over time

Organization admins can visualize usage trends for each category in the **Usage & Alerts** tab. Select your deployment type for instructions:

<Tabs>
<Tab title="Multi-tenant Cloud">
1. In the top right corner, click your user profile icon.
1. In the **Account** section, click **Usage & Alerts**.
1. From the sidebar, select a category to display its **Usage over time** chart.
</Tab>
<Tab title="Dedicated Cloud / Self-Managed">
1. In the top right corner, click your user profile icon.
1. Click **Organization Dashboard**.
1. Click **Usage & Alerts**.
1. From the sidebar, select a category to display its **Usage over time** chart.
</Tab>
</Tabs>

<Note>
Usage charts are visible to organization admins only. Team admins can't view them.
</Note>

> The categories in the sidebar depend on which products are active in your deployment, and can include **Storage**, **Tracked Hours**, **Weave**, **Inference**, **Training**, {/* Agent, */} and **Sandboxes**.

Reviewer: Casing check: this lists **Tracked Hours** (title case) while the prose at lines 109 and 140 uses "tracked hours" (lowercase). If "Tracked Hours" is the exact sidebar label, the split (UI label vs. common noun in prose) is defensible — just confirm it's intentional and matches the UI.

Author: This is the only one that's a UI label

To export usage data, click the download icon in the top right corner of the chart. The export returns a monthly usage breakdown by team and project for the last 24 months, regardless of the selected time range.

#### Current cycle compared to historical views

The **Current cycle** view is *cumulative*: each bar adds to the previous one to show your running total usage since the start of the current billing period. Use this view to see how your usage is pacing against its allowance. Granularity depends on the length of your billing cycle. For example, a monthly cycle shows daily bars and an annual cycle shows monthly bars.

The **Last 30 days**, **Last 90 days**, **Last 12 months**, and **Last 24 months** views are *historical*: each bar shows the usage for a single day or month rather than a running total. Use these views to compare usage across past periods. Because the top five teams are recalculated for the selected range, the colored teams can differ from those in the current cycle.

> Storage is always cumulative, because it measures the total volume currently stored (a 30-day weighted average, in GB) rather than usage consumed during a period.

Reviewer: This describes Storage as "the total volume currently stored (a 30-day weighted average, in GB)," but the table row at line 102 calls it "Total accumulated volume … measured in GB." "Currently stored" and "30-day weighted average" also pull in slightly different directions. The ticket flagged the storage/cumulative nuance as important to get right — can you confirm the precise phrasing against the billing code so the two descriptions agree?

Reviewer 2: It is kind of a combination of both which makes it slightly confusing. 
Storage is visualized as cumulative, so "accumulated volume" is correct verbiage here. But for each bar in the cumulative view, the value is a 30-day weighted average for the dates for that bar.
The verbiage here in line 74 is correct, and the only change I would suggest is making the text in the chart below for storage as: "measured as a 30-day weighted average, in GB".

{/* Source: https://wandb.ai/account-settings/wandb/alerts/storage (multi-tenant, Last 12 months, All teams) */}
<Frame>
    <img src="/images/app_ui/billing_usage_chart_12_months.png" alt="Storage usage over time chart for the last 12 months, showing one bar per month segmented by team" />
</Frame>

#### Stats bar and threshold lines

> In the **Current cycle** and **All teams** view, a stats bar above the chart summarizes **Usage**, **Allowance**, and **Daily Average Usage**. Daily Average Usage isn't shown for Storage.

Reviewer 2: We won't be showing the stats bar for Storage at all since we're not supporting current cycle for Storage

Reviewer 2: <img width="1484" height="344" alt="Image" src="https://github.com/user-attachments/assets/b9ddaab5-b5cb-4074-970d-391af17b95e9" />
Should also mention somewhere that we will be showing a "Total usage" value for historical views (non current cycle), and that total usage is the aggregated total usage for that specific view cycle

> In the **Current cycle** and **All teams** view, a stats bar above the chart summarizes **Usage**, **Allowance**, and **Daily Average Usage**. Daily Average Usage isn't shown for Storage.

Author: ```suggestion
In the **Current cycle** and **All teams** view, a stats bar above the chart summarizes **Usage**, **Allowance**, and **Daily Average Usage**. Storage charts do not display a stats bar or daily average usage; instead, historical views show the aggregated total usage for that specific time range.
```

> In the **Current cycle** and **All teams** view, a stats bar above the chart summarizes **Usage**, **Allowance**, and **Daily Average Usage**. Storage charts do not display a stats bar or daily average usage; instead, historical views show the aggregated total usage for that specific time range.

Reviewer: Nit: "the **Current cycle** and **All teams** view" reads awkwardly. Making the dual-selection condition explicit is clearer:

```suggestion
When both **Current cycle** and **All teams** are selected, a stats bar above the chart summarizes **Usage**, **Allowance**, and **Daily Average Usage**. Storage charts do not display a stats bar or daily average usage; instead, historical views show the aggregated total usage for that specific time range.
```

When a category has a defined allowance, the chart displays two threshold lines: a gold line at 85% of the allowance and a red line at 100%. These lines mark where [usage alerts](#usage-alerts) are sent to admins. Usage can exceed these thresholds. The lines are visual guides only.

{/* Source: https://wandb.ai/account-settings/wandb/alerts/training (multi-tenant, Current cycle, All teams; category with a defined allowance) */}
<Frame>
    <img src="/images/app_ui/billing_usage_chart_thresholds.png" alt="Usage over time chart showing a gold 85% threshold line and a red 100% threshold line, with cumulative usage exceeding the allowance" />
</Frame>

For categories with no usage limit, **Allowance** displays as **Unlimited** and no threshold lines appear.

<div id="usage-alerts">
### View and manage usage alerts

> </div>

Reviewer: Optional: Mintlify supports native custom anchors (`### Heading {#custom-id}`), which preserves the `#usage-alerts` anchor without the `<div>` wrapper. The wrapper is a localization-tooling pattern that otherwise appears only in `fr/ja/ko` files (and there as `##` indented two spaces). Renders fine as-is, so this is polish, not a blocker — worth a quick preview check since no English page uses `{#...}` yet.

```suggestion
### View and manage usage alerts {#usage-alerts}
```

| Category      | Description |
|---------------|-------------|
| Storage       | Total accumulated volume of your organization's stored artifacts and run data, measured in GB. |
| Weave         | Total volume of data imported during the billing period, measured in MB. Shown only if Weave is active in the deployment. |
| Inference     | Total input and output tokens used by Inference. See [Inference Pricing](https://wandb.ai/site/pricing/inference/). Shown only if Inference is active in the deployment. |
| Training      | Token-based inference and compute, measured in GPU/hour. See [Serverless RL Pricing](https://wandb.ai/site/pricing/reinforcement-learning/). Shown only if Serverless RL is active in the deployment. |
| Sandboxes     | Flat rate per billing period, determined by your plan. See [Pricing](https://wandb.ai/site/pricing/). Shown only if Sandboxes are active in the deployment. |

> {/* Agent | TODO | */}

Reviewer: Nit: leftover `{/* Agent | TODO | */}` placeholder (also at line 136). Harmless since it's a comment, but consider tracking the Agent SKU in a ticket rather than an inline TODO. When you enable it, confirm the label matches the UI — your PR description says "WB Agent," the comment says "Agent."

Author: It's on purpose since Agents is coming very soon and this shows me what to update. :)

For organizations on the **Enterprise** plan, all **Organization admins** and **Billing admins** receive and can configure usage alerts. On the [Pro plan](https://wandb.ai/site/pricing/), only the **Billing admin** receives usage alerts.

Messages are sent from `support@wandb.com`. The thresholds are not configurable, but you can selectively turn off one or both alerts per category.

To configure usage alerts for your organization, you must be an **Organization admin** or **Billing admin**.

1. In the top right corner, click your user profile icon.
1. In the **Account** section, click **Usage & Alerts**. Alternatively, click **Settings**, then click the **Usage & Alerts** tab.
1. From the sidebar, select a category.
1. Click the **Usage** tab to view your organization's usage and alert settings for that category.
1. Turn on or off the **85% usage alert** or **100% usage alert** for that category. <Warning>With no active usage alerts for a category, your organization could reach 100% of usage in a category without warning.</Warning>

<div id="spending-alerts">
### View and manage spending alerts

> </div>

Reviewer: Same as the usage-alerts heading — native custom anchor avoids the `<div>` wrapper while keeping `#spending-alerts`.

```suggestion
### View and manage spending alerts {#spending-alerts}
```
